packages/grafana-ui/src/components/Select/getSelectStyles.ts

```diff
@@ -15,6 +15,7 @@ export const getSelectStyles = stylesFactory((theme: GrafanaTheme2) => {
       minWidth: '100%',
       overflow: 'hidden',
       zIndex: 1,
+      padding: theme.spacing(0.5),
```

Comment:
think we should put this `padding` on the `ScrollContainer` in `SelectMenu` instead, else:
- the scroll indicator (shadow) doesn't properly span the full width of the menu
- you can't fully scroll the bottom element into view